### PR TITLE
Enable quarkus native compile feature

### LINKED ISSUE #11: Provide native binary of kafkaproxy for cloud native usage
Currently the docker images are graalvm based. These images are very huge and memory consuming. Also the startup-time with about one second is very high for cloud native usage.

Since kafkaproxy app is already based on the quarkus framework it's possible to create a native image to reduce the memory consumption about ten times. Also the startup times could be in regions about 50 ms.

### PR DESCRIPTION
**Pull request by alfsch**
- upgrade to quarkus 1.3.1-Final
- added application.properties to control native build features
- added docker native image build
- enabled quarkus auto service loader feature for native compile
- made Helpers class runtime initialized
- added native ssl feature
- gitignore eclipse and maven stuff

Closes #11

### CODE REVIEW

File: docker/src/main/docker/Dockerfile.linux.multistage [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## Stage 1 : cope all later needed files from image with native capabilities
```
Comment (dajudge) [inline docker/src/main/docker/Dockerfile.linux.multistage:1]: The first stage doesn't seem to do anything?

File: example/docker-compose.yml [hunk 2]
```diff
@@ -33,4 +33,6 @@ services:
     environment:
       KAFKAPROXY_HOSTNAME: localhost
       KAFKAPROXY_BASE_PORT: 4000
-      KAFKAPROXY_BOOTSTRAP_SERVER: kafka1:9092
\ No newline at end of file
+      KAFKAPROXY_BOOTSTRAP_SERVER: kafka1:9092
+# Uncomment to get trace logging
```
Comment (dajudge) [inline example/docker-compose.yml:37]: Please remove this comment (not true anymore anyway?)

File: README.md [hunk 3]
```diff
@@ -39,27 +39,32 @@ If you have `docker-compose` installed you can try out kafkaproxy by using the d
 `example` directory. So clone the [kafkaproxy repo](https://github.com/dajudge/kafkaproxy) and run the following commands:
 
 **Step 1:** Start kafka, zookeeper and kafkaproxy.
+
```
[the remark this answers is not in the document]
Comment (alfsch) [reply README.md]: done

File: docker/src/main/docker/Dockerfile.linux.multistage (hunk 1 above)
Comment (alfsch) [reply docker/src/main/docker/Dockerfile.linux.multistage]: not needed anymore

File: example/docker-compose.yml (hunk 2 above)
Comment (alfsch) [reply example/docker-compose.yml]: done

**Pull request merged**